pkg/types/conf.go

```diff
@@ -184,41 +184,24 @@ func mergeCNIRuntimeConfig(runtimeConfig *RuntimeConfig, delegate *DelegateNetCo
 // CreateCNIRuntimeConf create CNI RuntimeConf for a delegate. If delegate configuration
 // exists, merge data with the runtime config.
 func CreateCNIRuntimeConf(args *skel.CmdArgs, k8sArgs *K8sArgs, ifName string, rc *RuntimeConfig, delegate *DelegateNetConf) (*libcni.RuntimeConf, string) {
```

Comment:
Why you make into two function: CreateCNIRuntimeConf and CNIRuntimeConf?

To me, CreateCNIRuntimeConf seems to be redundant because it is just a one line.

```
func CreateCNIRuntimeConf(args *skel.CmdArgs, k8sArgs *K8sArgs, ifName string, rc *RuntimeConfig, delegate *DelegateNetConf) (*libcni.RuntimeConf, string) {
	return CNIRuntimeConf(args.ContainerID, string(k8sArgs.K8S_POD_INFRA_CONTAINER_ID), string(k8sArgs.K8S_POD_NAME),  string(k8sArgs.K8S_POD_NAMESPACE), string(k8sArgs.K8S_POD_UID), args.Netns, ifName, rc, delegate)
}
```

Reply:
same as in https://github.com/k8snetworkplumbingwg/multus-cni/pull/771#discussion_r782236223